### DISCUSSION TITLE
How to compile?

### DISCUSSION THREAD

**Discussion by IngwiePhoenix** [leonidboykov/getmoe#37] [Q&A] [answered]
I am used to C and C++, where I'd just compile all into one binary and put it into my `$PATH` - but Go is a little different. In fact, I don't know how to properly compile this project and get me an executable out of it :) Could you share that information? It might be useful for other users, too.

**Comment by leonidboykov** [comment, marked as answer]
Hello,

I have to install Go and run `go get github.com/leonidboykov/getmoe` or clone the project and run `go build ./cmd/getmoe/`

You may download binary for your OS from [releases](https://github.com/leonidboykov/getmoe/releases) 😉
